src/wallet/wallet2.cpp

```diff
@@ -6614,11 +6638,16 @@ bool wallet2::parse_unsigned_tx_from_str(const std::string &unsigned_tx_st, unsi
   s = s.substr(1);
   if (version == '\003')
   {
+    if (!m_load_deprecated_formats)
+    {
+      LOG_PRINT_L0("Not loading deprecated format");
+      return false;
+    }
     try
     {
       std::istringstream iss(s);
-      boost::archive::portable_binary_iarchive ar(iss);
-      ar >> exported_txs;
+      binary_archive<false> ar(iss);
```

Comment:
Is this this change accidental? This is the older version, which I think implies the file is using the previous format.

Reply:
Looks like it was, I'll revert.